modules/apps/object/object-test-util/src/main/java/com/liferay/object/test/util/ObjectDefinitionTestUtil.java

```diff
@@ -178,6 +204,37 @@ public static ObjectDefinition publishObjectDefinition(
 			objectDefinition.getObjectDefinitionId());
 	}
 
+	public static ObjectDefinition publishObjectDefinition(
+			boolean enableLocalization, List<ObjectField> objectFields,
+			User user)
+		throws Exception {
```

Comment:
Can you adapt the existing method: add the 3 parameter (user), and make the other usages pass `TestPropsValues.getUserId()` ?